File: API.Tools/ExercisesParser/Models/Exercise.cs
namespace ExercisesParser.Models;

public class Exercise
{
    public int Id { get; set; }
    public string Name { get; set; }
    public string Description { get; set; }
    public int Category { get; set; }
    public string CategoryName { get; set; }
    public List<int> Muscles { get; set; } = new List<int>();
    public List<int> MusclesSecondary { get; set; } = new List<int>();
    public List<int> Equipment { get; set; } = new List<int>();
    
    // Дополнительные поля которые могут быть полезны
    public string LicenseAuthor { get; set; }
    public List<string> Images { get; set; } = new List<string>();
    public List<string> Videos { get; set; } = new List<string>();
}

File: client/src/app/features/tabs/features/exercise-diary/types/Exercise.ts
import {ExerciseTypeEnum} from "./ExerciseTypeEnum";
import {ExerciseMeasurementEnum} from "./ExerciseMeasurementEnum";
import {MuscleTypeEnum} from "./MuscleTypeEnum";
import {EquipmentTypeEnum} from "./EquipmentTypeEnum";

export type Exercise = {
  id: string;
  name: string;
  description?: string;
  type: ExerciseTypeEnum;
  measurement: ExerciseMeasurementEnum;
  muscles: MuscleTypeEnum[];
  equipments: EquipmentTypeEnum[];
}
